test/test_helper.exs

```diff
@@ -2,6 +2,9 @@
 # Needs to be called before ExUnit.start
 {:ok, _} = Application.ensure_all_started(:ex_machina)
 
+# By default, exclude S3 tests
+# To run S3 tests, run `mix test --include s3`
+ExUnit.configure exclude: [:s3]
```

Comment:
I'm a little concerned about making this the default. I'd like to go the other way I think, and add the `exclude` command in the `README`.

We almost want to encourage people to get their own AWS account, too, so they can actually play around with some of that if they'd like to.

What do you think? It's scary to me to rely on a default that ignores some tests.